src/main/java/com/google/api/codegen/transformer/csharp/CSharpModelTypeNameConverter.java

```diff
@@ -244,6 +239,10 @@ private TypeName getTypeNameForTypedResourceName(
 
   @Override
   public TypedValue getEnumValue(TypeRef type, String value) {
-    throw new UnsupportedOperationException("getEnumValue not supported by C#");
+    TypeName enumTypeName = getTypeName(type);
+    List<String> enumTypeNameParts = Splitter.on('+').splitToList(enumTypeName.getNickname());
```

Comment:
It's in [`getTypeName`](https://github.com/eoogbe/toolkit/blob/f83f2de6a51ee3aa38e3bfa4b364899e6e75c8e9/src/main/java/com/google/api/codegen/transformer/csharp/CSharpModelTypeNameConverter.java#L133).